rules/windows/process_creation/process_creation_win_manipulation_of_tokens_via_impersonate_binary.yml

```
@@ -0,0 +1,29 @@
+title: possible manipulation of tokens on a windows computers remotely detected via impersonate
```

Since the logic is based on detecting the execution it should
```suggestion
title: Impersonate Execution
```

---

Got it!